components/use-cases/advertising-platforms/advertising-platforms.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Advertising Platforms
+permalink: /components/use-cases/advertising-platforms/
+---
+
+* TOC
+{:toc}
+
+With this end-to-end flow you can bring into Keboola Connection data from one or more marketing channels of your choice (Facebook Ads, LinkedIn Ads, Google Ads, and Bing Ads). 
+After all the necessary tasks are performed on the data, you load the results into BigQuery. You can also enrich the data with Google Analytics.
```

Comment:
chybí mi tady ten vlastní use-case, když už je to teda dokonce i v názvu! Proč bych to měl dělat tohle, co se například dozvím?